README.md

```diff
@@ -250,3 +250,19 @@ To set the multipart threshold with a boto3 client, do the following:
 config = TransferConfig(multipart_threshold=5*1024*1024*1024)
 self.boto3_client.upload_file(put_filename, bucket_name, key, Config=config)
 ```
+
+## Running Unit Tests
+
+Run the following commands to run the unit test suite.
```

Comment:
I feel like we need to do one or more of the following:

1. Indicate that the `docker-compose.yaml` file is in `tests/docker`
2. Make a separate README for how to run the tests in the `tests` directory

Thoughts?

Reply:
That sounds reasonable. I don't think what's there is incorrect for 0.1.0 though.

Perhaps that becomes a 0.2.0 thing.

Reply:
I think it is incorrect. This README is sitting in the top level and if I run `docker compose` it's not going to run anything...

We need to use the `--project-directory` option or instruct the reader to `cd` into the appropriate directory.

Reply:
The first instruction for running the tests is `cd tests/docker` which puts you in the directory where the compose file is located.

Reply:
Oh lol I see it now. Well, that settles that. My bad.